### DISCUSSION TITLE
Getting a runtime error when trying to login using the sample application

### DISCUSSION THREAD

**Discussion by samyap4** [Web3Auth/Web3Auth#803] [Help]
Hello,  I am trying to set up a demo integration of using Auth0 with web3auth.  I followed this guide: https://web3auth.io/docs/guides/auth0 and used your react sample app to get off the ground.  After configuring everything in my web3auth dashboard and my auth0 dashboard I went ahead and updated all the configs (to the best of my knowledge) in the sample app to have it pointed to my auth0 app/google social connection.  When I build I am getting two webpack errors that the user in the previous issue is seeing, but after skipping past those I am getting this error when trying to access the login screen: WalletLoginError: Failed to connect with walletFailed to login with openlogin.

I am not sure if fixing the build errors would resolve this or if there is something else I can try to get this integration up an running.  I tried googling and searching around but have not found anything yet.  TIA!

**Comment by shahbaz17** [comment]
Hey @samyap4 

Can you try to run this in a different browser or try removing the local & session storage from the Application tab of the developer menu inside the browser?

Also, please share your code snippet here.

**Reply by shahbaz17** [reply]
Also, here's our Auth0 example: https://github.com/Web3Auth/examples/tree/main/web-core-sdk/custom-authentication/auth0-react-core-example for reference.

**Comment by samyap4** [comment]
Hey @shahbaz17 the react polyfill build issue ending up fixing this issue as well, thank you to this team for the quick responses! Product seems very cool so far.
